### PR TITLE
Cf frm

### PR DESCRIPTION
**Pull request by abarret**

### CODE REVIEW

**Comment by abarret** [comment]
We should not merge this yet, because it requires a special build of IBAMR.

**Comment by abarret** [comment]
Once #66 is merged, we can merge this branch into `main`. Then we can finally stop doing what we said we wouldn't do again, which is put a massive amount of development in a different branch :smile:.

**Review by bindi-nagda — APPROVED** [review]

**Pull request merged**
